app/cdap/components/WranglerV2/SectionComponent/index.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
```

Comment:
change the copyright year to 2023 for all new files here and going forward

Reply:
changed copyright year to 2023